rero_ils/modules/local_entities/mappings/v7/local_entities/local_entity-v0.0.1.json

```diff
@@ -0,0 +1,76 @@
+{
+  "aliases": {
+    "unified_entities": {}
+  },
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "autocomplete": {
+          "type": "custom",
+          "tokenizer": "standard",
+          "filter": [
+            "lowercase",
+            "icu_normalizer",
+            "icu_folding",
+            "edge_ngram_filter"
+          ]
+        },
+        "search_autocomplete": {
+          "type": "custom",
+          "tokenizer": "standard",
+          "filter": [
+            "lowercase",
+            "icu_normalizer",
+            "icu_folding"
+          ]
+        }
+      }
+    }
+  },
+  "mappings": {
+    "date_detection": false,
+    "numeric_detection": false,
+    "properties": {
+      "autocomplete_name": {
+        "type": "text",
+        "analyzer": "autocomplete",
+        "search_analyzer": "search_autocomplete"
+      },
+      "authorized_access_point_sort": {
+        "type": "keyword",
+        "normalizer": "sort_normalizer"
+      },
+      "$schema": {
+        "type": "keyword"
+      },
+      "pid": {
+        "type": "keyword"
+      },
+      "type": {
+        "type": "keyword"
+      },
+      "preferred_name": {
+        "type": "text"
+      },
+      "authorized_access_point": {
+        "type": "text",
+        "copy_to": [
+          "autocomplete_name",
+          "authorized_access_point_sort"
+        ]
+      },
+      "organisations": {
+        "type": "keyword"
+      },
```

Comment:
seems strange.... I found it also on current `Entity` resource, but is it correct ? @jma @rerowep ?

Reply:
It is used here: https://github.com/rero/rero-ils/blob/420a39b88150562b506b0a7b22181a940099e94e/rero_ils/modules/entities/api.py#L171. To filter entities by viewcode. The field `organisations` should be renamed for example as `document_organisation_pids.

Reply:
ok